pennylane/ops/op_math/linear_combination.py

```diff
@@ -0,0 +1,464 @@
+import itertools
+import numbers
+from collections.abc import Iterable
+from copy import copy
+from typing import List
+
+import pennylane as qml
+from pennylane.operation import Observable, Tensor, Operator, convert_to_opmath
+
+from .composite import CompositeOp
+from .sum import Sum
+
+# LinearCombination class as special case of Sum
+
+OBS_MAP = {"PauliX": "X", "PauliY": "Y", "PauliZ": "Z", "Hadamard": "H", "Identity": "I"}
+
+
+def _compute_grouping_indices(observables, grouping_type="qwc", method="rlf"):
+    # todo: directly compute the
+    # indices, instead of extracting groups of observables first
+    observable_groups = qml.pauli.group_observables(
+        observables, coefficients=None, grouping_type=grouping_type, method=method
+    )
+
+    observables = copy(observables)
+
+    indices = []
+    available_indices = list(range(len(observables)))
+    for partition in observable_groups:  # pylint:disable=too-many-nested-blocks
+        indices_this_group = []
+        for pauli_word in partition:
+            # find index of this pauli word in remaining original observables,
+            for ind, observable in enumerate(observables):
+                if qml.pauli.are_identical_pauli_words(pauli_word, observable):
+                    indices_this_group.append(available_indices[ind])
+                    # delete this observable and its index, so it cannot be found again
+                    observables.pop(ind)
+                    available_indices.pop(ind)
+                    break
+        indices.append(tuple(indices_this_group))
+
+    return tuple(indices)
+
+
+class LinearCombination(Sum):
```

Comment:
We could have `LinearCombination` show up in [op_math](https://xanaduai-pennylane--5216.com.readthedocs.build/en/5216/code/qml_ops_op_math.html) docs pages rather than in the top-level `qml` docs (same for how it is imported).

Reply:
✅